app/assets/javascripts/lib/requests.js

```diff
@@ -1,8 +1,24 @@
 import Axios from 'axios';
 
+function destroy_modal(modal) {
+  if(modal){
+    modal.el.dataset.processing = "false";
+    modal.destroy();
+  }
+}
+
 export function request (url, method, data = {}, options = {scroll: true}) {
-  const csrf_el = document.querySelector('meta[name=csrf-token]');
-  let headers = csrf_el ? {'X-CSRF-Token': csrf_el.getAttribute('content')} : {};
+
+  // For now, separate handling of csrf in the Rails and Django apps.
+  const rails_csrf_el = document.querySelector('meta[name=csrf-token]');
```

Comment:
Django recommends getting the token from the cookie instead of from the DOM, though it's not super persuasive about why:

https://docs.djangoproject.com/en/2.2/ref/csrf/#acquiring-the-token-if-csrf-use-sessions-and-csrf-cookie-httponly-are-false

(see especially the Note, but then sort of undermined by the Warning). Any feeling here about why to prefer the DOM element in our case?

Reply:
10% I wasn't totally confident we'd stick with "CSRF_USE_SESSIONS and CSRF_COOKIE_HTTPONLY are False"; 10% I was lazy; 80% I was unduly influenced by the knowledge this particular case probably wanted to be a normal form submission in the future.

Happy to use the cookie instead.